### PR TITLE
Docker altnets

### PR DESCRIPTION
**Pull request by rkeene**
Support and automatically build docker containers for testing networks

### CODE REVIEW

**Comment by bradennapier** [comment]
Time to get some BaeBlocks!

File: docker/node/entry.sh [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+case "${network}" in
+        live|'')
+                network='live'
+                dirSuffix=''
+                ;;
+        beta)
+                dirSuffix='Beta'
+                ;;
+        test)
+                dirSuffix='Teta'
```
Comment (PlasmaPower) [inline docker/node/entry.sh]: Typo. Also are you sure the dir changes? I didn't think so.

Comment (rkeene) [reply docker/node/entry.sh]: For Beta atleast it does:

```
[<ROOT> bawt2 ci]$ docker container exec -it beta1 bash
root@1e3ee0231731:/# cd /root
root@1e3ee0231731:~# ls -l
total 0
drwxrwxr-x+ 3 root root 73 Mar  8 06:55 RaiBlocksBeta
root@1e3ee0231731:~# cd RaiBlocksBeta
root@1e3ee0231731:~/RaiBlocksBeta# ls -l
total 72
-rw-r--r--+ 1 root root  2404 Mar  8 06:55 config.json
-rw-------+ 1 root root 65536 Mar  8 07:00 data.ldb
-rw-------+ 1 root root  8192 Mar  8 16:56 data.ldb-lock
drwxrwxr-x+ 2 root root   117 Mar  8 13:45 log
root@1e3ee0231731:~/RaiBlocksBeta# exit
exit
[<ROOT> bawt2 ci]$
```

Comment (PlasmaPower) [reply docker/node/entry.sh]: IIRC it doesn't for test, but I'm not sure.

Comment (rkeene) [reply docker/node/entry.sh]: https://github.com/nanocurrency/raiblocks/blob/6051604c10a399e68effbf148a7fa359e0a1b277/rai/node/utility.cpp#L9-L25

Comment (PlasmaPower) [reply docker/node/entry.sh]: Nevermind then. Typo still needs fixed though.

Comment (rkeene) [reply docker/node/entry.sh]: Fixed the typo

**Comment by clemahieu** [comment]
Looks good to me.

**Pull request merged**